### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Charlotte-br560**
rebrand prodcut name updated (proxycrawl old name now Crawlbase )

## Requirements

 * [ ] This is Software as a Service not self hosted
 * [ ] It has a free tier not just a free trial
 * [x] Pricing information is clearly visible without signup or phone calls
 * [ ] The submission mentions what is free
 * [ ] The submission is not already present in the list
 * [ ] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -324,7 +324,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Preset Cloud](https://preset.io/) - A hosted Apache Superset service. Forever free for teams of up to 5 users, featuring unlimited dashboards and charts, a no-code chart builder, and a collaborative SQL editor.
   * [PromptLeo](https://promptleo.com/) - Prompt engineering platform for creators and developers. It offers a prompt engineering library, forms, and API. The free plan provides one prompt formation, one prompt API endpoint, and 30 generations per month.
   * [PromptLoop](https://www.promptloop.com/) - Use AI and large language models like GPT-3 with a simple spreadsheet formula to transform, comprehend, and analyze text in Google Sheets. The first 2,000 credits are free each month.
-  * [ProxyCrawl](https://proxycrawl.com/) — Crawl and scrape websites without proxies, infrastructure, or browsers. We solve captchas for you and prevent you from being blocked. The first 1000 calls are free of charge.
+  * [Crawlbase](https://crawlbase.com/) — Crawl and scrape websites without proxies, infrastructure, or browsers. We solve captchas for you and prevent you from being blocked. The first 1000 calls are free of charge.
```
Comment (ripienaar) [inline README.md:327]: We only lost things once. And listed entries have to meet all the criteria. Please close the ones that don’t and put it one entry only

Comment (madhuryadutta) [reply README.md:327]: @ripienaar ,  **crawlbase.com** is more like a **Free Trial** not a **Free tier**. The 1000 request is fixed , It does not reset monthly.   Extra 9000 will be provide on adding Credit card which is definitely a Business Strategy , 

**Same Pull Request** ::   #3206 #3207  


Screenshot from 2023-12-28 21-53-27
Their **Documenation** also mention it as a **Free Trial**

Screenshot from 2023-12-28 21-53-22

Comment (ripienaar) [reply README.md:327]: Ok. Then we can close all these

Comment (ripienaar) [reply README.md:327]: Thank you @madhuryadutta

Comment (madhuryadutta) [reply README.md:327]: you're welcome :+1:

**Pull request closed without merging**